src/Util/PhpServicesCreator.php

```diff
@@ -0,0 +1,639 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Util;
+
+use PhpParser\Builder\Param;
+use PhpParser\BuilderFactory;
+use PhpParser\Node;
+use PhpParser\PrettyPrinter\Standard;
+use Symfony\Bundle\MakerBundle\Str;
+use Symfony\Component\Yaml\Parser;
+use Symfony\Component\Yaml\Tag\TaggedValue;
+
+/**
+ * This class allows to parse and convert the YAML service configuration file to PHP format.
+ *
+ * @author Antoine Michelet <jean.marcel.michelet@gmail.com>
+ *
+ * @internal
+ */
+final class PhpServicesCreator
+{
+    private $factory;
+    private $creatorFactory;
+    private $stmts = [];
+    private $useStatements = [];
+
+    public function __construct()
+    {
+        $this->factory = new BuilderFactory();
+    }
+
+    public function convert(string $yamlFile): string
+    {
+        $this->creatorFactory = $this->factory->namespace(
+            'Symfony\Component\DependencyInjection\Loader\Configurator'
+        );
+
+        $closureStmt = $this->createClosureStmts((new Parser())->parse($yamlFile, 768));
```

Comment:
768 won't do :). You need to use a bitwise operator with the `Yaml` constants

Reply:
```suggestion
        $closureStmts = $this->createClosureStmts((new Parser())->parse($yamlFile, 768));
```

Reply:
Done.